test/protocol/building/auth_packet_test.c

```diff
@@ -132,6 +163,36 @@ TEST test_build_auth_packet_sha2_with_pass()
     PASS();
 }
 
+TEST test_build_auth_packet_cleartext_with_pass()
+{
+    trilogy_builder_t builder;
+    trilogy_buffer_t buff;
+    int err = trilogy_buffer_init(&buff, 1);
+    ASSERT_OK(err);
+
+    err = trilogy_builder_init(&builder, &buff, 1);
+    ASSERT_OK(err);
+
+    static const char scramble[] = "I/JiR>}{tU-{G3e$\"{hx";
+
+    err = trilogy_build_auth_packet(&builder, user, pass, strlen(pass), NULL, 45, "mysql_clear_password", scramble, 0);
+    ASSERT_OK(err);
+
+    static const uint8_t expected[] = {
```

Comment:
Should there be a cleartext password in this expected output? I think it's the same as the same bytes as in test_build_auth_packet_cleartext (although a little difficult to visually diff).